openedx_external_enrollments/settings/common.py

```diff
@@ -68,3 +68,12 @@ def plugin_settings(settings):
     settings.DROPBOX_API_ARG_UPLOAD = '{"path":"%s","mode":{".tag":"overwrite"}}'
     settings.DROPBOX_API_UPLOAD_URL = "/files/upload"
     settings.DROPBOX_DATE_FORMAT = "%m-%d-%Y %H:%M:%S"
+    settings.ICC_API_TOKEN= "icc-api-token"
+    settings.ICC_BASE_URL = "https://icchas11.stage.kineoplatforms.net/webservice/rest/server.php"
```

Comment:
This setting should be on the production.py also

Reply:
Added into production.py file